rubicon_ml/viz/experiments_table.py

```diff
@@ -314,3 +351,43 @@ def update_selected_experiment_table_rows(
                 return experiment_table_indices
 
             return []
+
+        @self.app.callback(
+            Output("publish-modal", "is_open"),
+            [
+                Input("publish-selected-button", "n_clicks_timestamp"),
+                Input("publish-button", "n_clicks_timestamp"),
+            ],
+            [
+                State("publish-modal", "is_open"),
+                State("publish-path-input", "value"),
+                State("experiment-table", "derived_virtual_selected_rows"),
+                State("experiment-table", "derived_virtual_data"),
+            ],
+        )
+        def toggle_publish_modal(
+            last_publish_selected_click,
+            last_publish_click,
+            is_modal_open,
+            publish_path,
+            selected_rows,
+            data,
+        ):
+            last_publish_selected_click = (
+                last_publish_selected_click if last_publish_selected_click else 0
+            )
+            last_publish_click = last_publish_click if last_publish_click else 0
+
+            if last_publish_selected_click > last_publish_click:
+                return True
+            elif last_publish_click > last_publish_selected_click:
+                selected_experiment_ids = [data[row].get("id") for row in selected_rows]
+                selected_experiments = [
+                    e for e in self.experiments if e.id in selected_experiment_ids
+                ]
+
+                publish(selected_experiments, publish_path)
```

Comment:
also I noticed each publish overwrites the other, could we have it appended?

Reply:
we have #174 for updating catalogs - once thats done we could leverage it in here